src/ring_theory/polynomial/symmetric.lean

```diff
@@ -44,6 +44,15 @@ open equiv (perm)
 open_locale big_operators
 noncomputable theory
 
+namespace multiset
+
+variables {R : Type*} [comm_semiring R]
+
+/-- The `n`th elementary symmetric function evaluated at the elements of `s` -/
+def esymm (s : multiset R) (n : ℕ) : R := ((s.powerset_len n).map multiset.prod).sum
+
+end multiset
```

Comment:
If we wanted, we could define this for `list` too, using `list.sublists_len`.

Reply:
I guess we could but I am not sure what the sue would be. I am not really using the ```multiset``` version for anything in this file.